SiennaOpenAPIModels.jl/src/sienna_to_json/static_injections.jl

```diff
@@ -0,0 +1,22 @@
+function convert(thermal_standard::PSY.ThermalStandard, ids::IDGenerator)
```

Comment:
Can we rename this function to something more explicit?

Reply:
How's psy2openapi?